internal/bucket/object/lock/lock.go

```diff
@@ -411,14 +413,15 @@ func ParseObjectLockRetentionHeaders(h http.Header) (rmode RetMode, r RetentionD
 	if err != nil {
 		return rmode, r, ErrInvalidRetentionDate
 	}
+	_, replReq := h[xhttp.MinIOSourceReplicationRequest]
```

Comment:
```suggestion
	_, replReq := h[xhttp.MinIOSourceReplicationRequest]
```

Can you make this case insensitive? or atleast use `h.Get()` instead?

Reply:
done